namespaces/live.cloud-platform.service.justice.gov.uk/formbuilder-publisher-test/resources/publisher.tf

```diff
@@ -1,15 +1,18 @@
 module "publisher-rds-instance" {
   source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.14"
 
-  vpc_name                   = var.vpc_name
-  db_backup_retention_period = var.db_backup_retention_period
-  application                = "formbuilderpublisher"
-  environment-name           = var.environment-name
-  is-production              = var.is-production
-  namespace                  = var.namespace
-  infrastructure-support     = var.infrastructure-support
-  team_name                  = var.team_name
-  db_engine_version          = "10"
+  vpc_name                    = var.vpc_name
+  db_backup_retention_period  = var.db_backup_retention_period
+  application                 = "formbuilderpublisher"
+  environment-name            = var.environment-name
+  is-production               = var.is-production
+  namespace                   = var.namespace
+  infrastructure-support      = var.infrastructure-support
+  team_name                   = var.team_name
+  db_engine_version           = "12"
+  allow_major_version_upgrade = "true"
+  rds_family                  = "postgres12"
+  db_instance_class           = "db.t3"
```

Comment:
This will need to have a qualified size, e.g. `db.t3.large`

Reply:
Ah sorry - we're getting away with the default t2.small currently, is t3.small an option?

Reply:
Yep!